# maintainer:kbkn/mori
#
# name of the port that connects to gps device
dev_name: /dev/ttyACM0
# directory of the file where waypoints will be saved
output_file: /home/ubuntu/catkin_ws/src/igvc2023/config/gps_waypoints/gps_waypoints.yaml
#output_file: /home/ubuntu/catkin_ws/src/igvc2023/config/gps_waypoints/qualification_waypoints.yaml
# country_id: 0, then Japan
# country_id: 1, then USA
country_id: 0
# true north is 0 degrees, clockwise angle
azimuth: 180
# latitude and longitude information given by the organizer at IGVC
#  - - latitude(1)
#    - longitude(1)
#  - - latitude(2)
#    - longitude(2)
#      ............
latitude_longitude:
  - - 42.66792770
    - -83.21932764
  - - 42.66807663
    - -83.21935915
  - - 42.66812064
    - -83.21936060
  - - 42.66826972
    - -83.21934030


#  - - 42.66826972
#    - -83,21934030
#  - - 42.66812064
#    - -83.21936060
#  - - 42.66807663
#    - -83.21935915
#  - - 42.66792770
#    - -83.21932764
